### PR TITLE
feat: add dbconnector.yaml config file

### PR DESCRIPTION
**Pull request by s3lph**

### CODE REVIEW

File: roles/appsuite/templates/opt/open-xchange/etc/dbconnector.yaml.j2 [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    autoReconnect: false
+    useServerPrepStmts: false
+    useTimezone: true
+    serverTimezone: UTC
+    connectTimeout: {{ appsuite_dbconnector_connect_timeout }}
+    socketTimeout: {{ appsuite_dbconnector_socket_timeout }}
+    useSSL: false
+    requireSSL: false
+    verifyServerCertificate: false
+    enabledTLSProtocols: TLSv1,TLSv1.1,TLSv1.2
```
Comment (ghost) [inline roles/appsuite/templates/opt/open-xchange/etc/dbconnector.yaml.j2]: Could you parameterize those values and not hardcode them? Or will those values never be changed?

Comment (s3lph) [reply roles/appsuite/templates/opt/open-xchange/etc/dbconnector.yaml.j2]: So far we've only been parametrizing the values we actually needed to change, simply due to the sheer amount of config options that AppSuite has.
Chances are the other values in this file won't need changing anytime soon.

Comment (s3lph) [reply roles/appsuite/templates/opt/open-xchange/etc/dbconnector.yaml.j2]: They are now all templated with their former hardcoded values as defaults.

**Review by lukasgraesslin — APPROVED** [review]
LGTM

**Review by SaschaD-Adfinis — APPROVED** [review]
LGTM

**Comment by s3lph** [comment]
Released on Galaxy as 1.0.2

**Pull request merged**